apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/db/DbPreparedStatementWrapper.java

```diff
@@ -27,7 +27,7 @@ public boolean execute() throws SQLException {
         long start = System.currentTimeMillis();
         boolean b = super.execute();
         long elapsed = System.currentTimeMillis() - start;
-        if (elapsed > stmtThreshold)
+        if (elapsed > stmtThreshold && blockchain != null)
```

Comment:
In unit tests it possible, I got such errors, so let it stay here. It doesn't do much harm.

Reply:
I think, in such case we should update our unit tests instead of modifying source code

Reply:
I don't see much problem here, because it prevents NPE at least. 
Yes, there is no much chance when **blockchain** will be NULL in real running node code, but code prevents NPE if that will be a case unexpectedly.
We want have unit test as isolated as possible, so why it's better to put more components into unit test just for logging some debug messages?

Reply:
Ok, add comments, that blockchain can be null only in unit tests.